### ISSUE TITLE
Getting started doesn't make it quick to get to the installer

### ISSUE THREAD

**Issue by rodet** [ibm-cloud-docs/cli#132]
1. What are the details of the issue you're experiencing?

Having worked recently with other cloud tools, I couldn't help but notice how the installer link is lost on the page. Even then, it is not a direct link to the installer, the user have to manually choose a platform. Other cloud providers make it much simpler to install the tool by taking a cross-platform approach (for example with NPM) - the key thing is that developers can install the tool in one line.
NPM would probably not work for IBM Cloud, but still the browser app should be intelligent and preselect the platform in the context and give the download file as a big icon that detaches from the rest. Linking a github repository at that point is kinda “neat” but in a bad way. It's already the second link on my way to find an installer that should have been available on one click.
Same for the supported OSes. Good to know what is supported, but most websites today answer from the browser agent response. If someone wants to install for a whole org somehow, this might be a separate information. I haven’t seen much of supported information in the way of the first use in many cases of the development tools I use on a regular basis. Overall the page is very overloaded and very hard to parse. Going through multiple pages, each time searching for information, is very detrimental to a step that's a no-brainer for most competitors.

2. Is this issue in reference to a specific page? If so, please provide a link.
https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli

3. Do you have a suggestion about how to resolve the issue?
- Possibly distribute the installer directly from IBM Cloud, preselect the user platform by default, while allowing for another platform in case this is for another system. Many online app downloads provide this feature by default nowadays.
- Clearly advocate for one approach of installing, instead of having two equal ones. What is the difference between shell and installer? Pro and cons? How can I decide on one or the other?
- You want the link to the installer (from the right platform) to stand out from all the rest and be a direct download, because Cloud Foundry directly link to here for downloading the installer.
- Put the remaining information in the background. Supported platform might be useful, no doubts, but it's less important that providing a download quick.

**Comment by christoc-ibm** [comment]
I created an internal dev issue to replace this external issue that was created in the doc repo. Thank you!

https://github.ibm.com/ibmcloud-cli/bluemix-cli/issues/3920
